packages/compass-aggregations/src/modules/pipeline-builder/stage.ts

```diff
@@ -97,7 +84,6 @@ export default class Stage {
         this.node.properties[0].value = babelParser.parseExpression(value);
       }
       assertStageNode(this.node);
-      assertStageValue(value);
```

Comment:
`mongodb-query-parser` just calls `ejson-shell-parser` internally which doesn't do any additional validation beyond the one we already did by parsing expression with babel parser above